src/main/java/org/vaadin/example/GreetService.java

```diff
@@ -8,7 +8,7 @@
 @VaadinSessionScoped
 public class GreetService {
 
-    public String greet(String name) {
+    public static String greet(String name) {
```

Comment:
isn't this supposed to be non-static because it is injected using CDI?

Reply:
I don't know why my changes have been removed. I fixed it.